### ISSUE TITLE
Non-rules enforced "manual" mode?

### ISSUE THREAD

**Issue by TheOrgg** [buttonmen-dev/buttonmen#2876]
So is it possible to have a mode where you either pick your button or use Custom BM and then don't have the AI control what happens?

You click on dice on each side, type in what you're doing into the chat, then use two dropdowns to select what happens to it?  Reroll, Score Die, die falls asleep, half die's size, grow/shrink by 1, etc?  Then there's a checkbox that says 'retain turn' just in case there's a trip attack or what have you.  Maybe a place to manually calculate a running score?

I was thinking this could be used to test buttons that use skills that aren't yet active on the site.

**Comment by blackshadowshade** [comment]
It sounds like what you want is a sandbox mode where you control everything that you want to, similar to #523. Is this the same idea?

**Comment by TheOrgg** [comment]
Yes.  I was talking to my friend I meet with to play BM occasionally, and he was thinking of an Outlaw Star set, but didn't want to constrain it to Buttonweavers-finished skills.
